thamos/cli.py

```diff
@@ -1367,10 +1369,24 @@ def images(
                 key = _REPORT_TRANSLATION_TABLE_IMAGES.get(
                     key, key.replace("_", " ").capitalize()
                 )
+                if key == "Container image analysis":
```

Comment:
Could this link be constructed before using the translation table? It might be more clear and less prone to errors when we adjust the string later.